Remotion/ObjectBinding/Web/UI/Controls/BocList.cs

```diff
@@ -1886,7 +1903,9 @@ public BocListSortingOrderEntry[] GetSortingOrder ()
             }
             else
             {
-              var newIndex = Array.IndexOf(columns, entry.Column);
+              var newIndex = entry.Column is BocColumnDefinition bocColumnDefinition
```

Comment:
given the old semantic, wouldn't this now be `columns.IndexOf((BocColumnDefinition)entry.Column)` ?